src/plugins/kibana_react/public/page_template/no_data_page/no_data_card/elastic_agent_card.tsx

```diff
@@ -35,10 +35,24 @@ export const ElasticAgentCard: FunctionComponent<ElasticAgentCardProps> = ({
     services: { http, application },
   } = useKibana<CoreStart>();
   const addBasePath = http.basePath.prepend;
-  const image = addBasePath(`/plugins/kibanaReact/assets/elastic_agent_card.svg`);
+  const imageUrl = addBasePath(`/plugins/kibanaReact/assets/elastic_agent_card.svg`);
   const canAccessFleet = application.capabilities.navLinks.integrations;
   const hasCategory = category ? `/${category}` : '';
 
+  const image = (
+    <EuiImage
+      size="fullWidth"
+      style={{
+        width: `max(100%, 360px)`,
+        height: 240,
+        objectFit: `cover`,
+        background: `aliceblue`,
```

Comment:
Just a nit: I don't think you need backticks here, regular '' should work.

Reply:
Yeah, usually my IDE converts these on save, wonder why it didnt. Fixed [2a33179](https://github.com/elastic/kibana/pull/132376/commits/2a33179b30900b4dc64cc675d21cd837950acb9c)